articles/cognitive-services/Face/QuickStarts/Python.md

```diff
@@ -4,317 +4,141 @@ description: Get information and code samples to help you quickly get started us
-* Facial attributes including age, gender, smile intensity, head pose, and facial hair. 
+## Prerequisites
 
-#### Face Detect python example request
+You must have a [Cognitive Services API account](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account) with **Face APIs**. The [free trial](https://azure.microsoft.com/en-us/try/cognitive-services/?api=face-api) is sufficient for this quickstart. You need the access key provided when you activate your free trial, or you may use a paid subscription key from your Azure dashboard.
+
+To continue with this walkthrough, please replace `subscription_key` below with a valid subscription key that you obtained earlier.
 
-1. Copy the appropriate section for your version of Python and save it to a file such as `detect_faces.py`.
-1. Replace the `subscriptionKey` value with your valid subscription key.
-1. Change the `uri_base` value to use the location where you obtained your subscription keys.
-1. Run the sample.
 
 ```python
-import httplib, urllib, base64, json
-
-###############################################
-#### Update or verify the following values. ###
-###############################################
-
-# Replace the subscription_key string value with your valid subscription key.
-subscription_key = '13hc77781f7e4b19b5fcdd72a8df7156'
-
-# Replace or verify the region.
-#
-# You must use the same region in your REST API call as you used to obtain your subscription keys.
-# For example, if you obtained your subscription keys from the westus region, replace 
-# "westcentralus" in the URI below with "westus".
-#
-# NOTE: Free trial subscription keys are generated in the westcentralus region, so if you are using
-# a free trial subscription key, you should not need to change this region.
-uri_base = 'westcentralus.api.cognitive.microsoft.com'
-
-# Request headers.
-headers = {
-    'Content-Type': 'application/json',
-    'Ocp-Apim-Subscription-Key': subscription_key,
-}
+subscription_key = "354f9ac5f2f04a008504575c15f458a7"
+assert subscription_key
+```
+
+Next, please verify the URL below and make sure it corresponds to the region you used when generating the API key. If you used a trial subscription, you are good to go!
+
+
+```python
+face_api_url = 'https://westcentralus.api.cognitive.microsoft.com/face/v1.0/detect'
+```
+
+Here is the URL of the image above. You can experiment with different images simply by changing ``image_url`` to point to a different image and re-running this notebook.
+
+
+```python
+image_url = 'https://how-old.net/Images/faces2/main007.jpg'
+```
+
+We next call into the Face API to recognize the faces in the image. In this instance, the image is specified via a publically-visible URL. You can also pass an image directly as part of the request body. For more details, please refer to the [API documentation](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395236). 
```

Comment:
s/recognize/detect/g

Please ensure to know the difference between Detection and Recognition. They are totally different.

Reply:
Thanks for pointing this out - fixed!